translator/test/src/test/resources/features/TranslatorUnitTests.feature

```diff
@@ -152,4 +152,171 @@ Feature: Translator Service
     Given I create kura data message with channel with scope "kapua-sys", client id "rpione3" and payload with body and metrics
     And I try to translate kura data message to mqtt message
     Then I get mqtt message with channel with scope "kapua-sys", client id "rpione3" and non empty body
-    And No exception was thrown
\ No newline at end of file
+    And No exception was thrown
+
+    #TranslatorDataMqttKura
+
+  Scenario: Translating of mqtt message with invalid payload and invalid topic into kura data message
+  Creating mqtt message with invalid payload and invalid topic that contain accountName only. Trying to translate it into kura data message.
+  Invalid channel exception should be thrown.
+
+    Given I create mqtt message with valid payload "invalidPayload" and invalid topic "kapua-sys"
+    And I expect the exception "InvalidChannelException" with the text "Invalid channel: kapua-sys"
+    When I try to translate mqtt message to kura data message
+    Then An exception was thrown
+
+  Scenario: Translating of mqtt message with invalid payload and with null topic into kura data message
+  Creating mqtt message with invalid payload and invalid "null" MqttMessage. Trying to translate it into kura data message.
+  Invalid message exception should be thrown.
+
+    Given I create mqtt message with valid payload "invalidPayload" and valid topic "kapua-sys/rpione3/DEPLOY-V2/GET"
+    And I expect the exception "InvalidMessageException" with the text "Invalid message: null"
```

Comment:
Maybe this is something I need  to check. 
There should be a meaningful message